### DISCUSSION TITLE
CEXs Deposit Wallets

### DISCUSSION THREAD

**Discussion by Tomwhh** [duneanalytics/spellbook#3047] [Sector: Labels]
Gm sers, I have an idea for labeling all the CEXs deposit addresses. I would love to run this idea & logic and see if it makes sense before executing it.

WHY:
When depositing funds to CEXs, users do not send them to CEXs' wallets directly. Instead, it is through intermediary wallets, Deposit Wallets. Without the cex deposit labels, we might not have a clear view of the final destination and flow of the transaction. The current approach is when I saw a transaction on Dune, I will need to go to etherscan and look into the next transaction of that deposit wallet to understand which exchange the entity is depositing into.

Logic:
- Only have outflow to a single exchange 
- Leveraging the cex labels for identification

Schema
- Similar to the cex label one

All feedback appreciated

**Comment by Tomwhh** [comment]
cc: @hildobby, @andrewhong5297 & @soispoke for feedback

**Comment by andrewhong5297** [comment]
Hey sorry I'm responding so late here. It would def be great to try and label these wallets - do you have an example query where you've explored this?

**Reply by Tomwhh** [reply]
@andrewhong5297 here is a sample query for getting the Coinbase Deposit Wallets
In total, there are 461k wallets based on the logic. Also did a quick sanity check of 10 wallets, which matches the labels on metadock as a Coinbase Deposit wallet. Love to have your feedback and thought ser! Thank you in advance

https://dune.com/queries/2380552

**Reply by andrewhong5297** [reply]
got it - yeah this seems like a natural next label to build cc: @hildobby @MSilb7 have probably looked into this before. 

I wonder what category this kind of label would go under, thinking something related to "funding" as people could get tokens from payments from stuff like utopia/safes, grants, CEXs, etc. idk if that is too much of a stretch though
